store/store.go

```diff
@@ -596,7 +596,7 @@ func (s *SnapUbuntuStoreRepository) ListRefresh(installed []*RefreshCandidate, a
 	// build input for the updates endpoint
 	jsonData, err := json.Marshal(metadataWrapper{
 		Snaps:  currentSnaps,
-		Fields: []string{"snap_id", "package_name", "revision", "version", "download_url"},
+		Fields: []string{"snap_id", "package_name", "revision", "version", "download_url", "origin"},
```

Comment:
Is the `origin` field the old name of the developer field?
Should we add a TODO to get the store to rename this term on their side?